<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoUpdateWidget</class>
 <widget class="QWidget" name="AutoUpdateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="8,1,1">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>正在检查更新。。。</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnDownload">
        <property name="text">
         <string>下载</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancle">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
